### ISSUE TITLE
More ceiling observation rooms depending on style

### ISSUE THREAD

**Issue by Flametusk** [TeamSpen210/BEE2-items#1106]
Most styles for the ceiling observation rooms use the regular light grid that clean uses, with very little variation. The only one that's noticeably different is the p1 ones.
I think that there should be more, depending on style (For example, overgrown could show the sky)

**Comment by LautaroL20** [comment]
This definitelly should be a separated item. The current observations takes a considerable ammount of space, so, i do not want them to keep using more.

**Comment by Flametusk** [comment]
Yeah, and some don't provide nearly enough light, (all except portal 1), and, say I want to highlight a 3x3 area, in portal 1 all I need to do is put recess the light 1 into the ceiling. That makes the little area 3 by 3, with a bit of a sharper cutoff, and it still looks normal, due to the big light already being recessed.
